components/common/CommonDetail/CommonDetail.tsx

```diff
@@ -2,40 +2,51 @@
 
 import { ButtonStyle, ButtonSize } from "types/enums/button.enum";
 import { CommonLayout, CommonBtn } from "components/common";
+import { useRouter } from "next/navigation";
 import styles from "./CommonDetail.module.scss";
 
-type Detail = "EMPLOYER" | "EMPLOYEE";
+type Detail = "EMPLOYER" | "EMPLOYEE" | "APPLICATIONDETAILS";
+
+interface Ias {
```

Comment:
우선 `IAs`로 인터페이스 네이밍 컨벤션 지켜주시고, 어떤 타입인지 이름만으로 불명확한것 같습니다.. 
```suggestion
interface IAs {
```

Reply:
죄송합니다 제가 수정을 안했네요 ㅠ